volatility3/framework/plugins/windows/devicetree.py

```diff
@@ -0,0 +1,145 @@
+# This file is Copyright 2022 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+
+from typing import Iterator, List, Tuple
+
+from volatility3.framework import constants, renderers, exceptions, interfaces
+from volatility3.framework.configuration import requirements
+from volatility3.framework.renderers import format_hints
+from volatility3.plugins.windows import driverscan
+
+DEVICE_CODES = {
```

Comment:
It might be a bit more work, and I'm ok with leaving this as it is for the moment, but it might be more useful to others to add this to an ISF JSON file as an enumeration?  Ideally it could be tied directly in to the `DEVICE_OBJECT` but I think overriding a single member of a structure isn't really feasible (and probably isn't worth it).

At the moment the list is short enough that I'm ok having it embedded in this file.

Reply:
That's right, if I could expand `_DEVICE_OBJECT` Symbol, I would have considered that option, but I put it in the code because I thought it was impossible with the current method.
If there are any changes or updates in the future, I will try to improve it.